runtests

```diff
@@ -19,7 +19,10 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
 
 import argparse
-import configparser
+try:
+    import configparser
+except ImportError:
+    import ConfigParser as configparser
```

Comment:
I don't think this commit is necessary. We only support Ubuntu Make with python3, not pythoN2, so you can remove this try/except statement.